x-pack/legacy/plugins/monitoring/server/routes/api/v1/apm/metric_set_overview.js

```diff
@@ -40,34 +40,60 @@ export const metricSet = [
   },
   {
     keys: [
-      'apm_responses_count',
-      'apm_responses_valid_ok',
-      'apm_responses_valid_accepted'
+      'apm_server_responses_count',
+      'apm_server_responses_valid_ok',
+      'apm_server_responses_valid_accepted',
+      'apm_server_responses_errors_count',
     ],
-    name: 'apm_responses_valid'
+    name: 'apm_server_responses_valid'
```

Comment:
Should the name be changed into `apm_server_responses` now that it also contains `apm_server_responses_errors_count`? (Same for acm).